tests/test_wallpaper.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from uiplib.Wallpaper import Wallpaper
+import tempfile
+
+
+class WallpaperTest(unittest.TestCase):
+
+    def test_wallpaper(self):
+        wallpaper = Wallpaper()
+        current_wallpaper = wallpaper.get()[7:]
```

Comment:
Maybe someday we'll need a `file://` so I left it that way. 

But since we can change it again I'll fix it.